_authors/osci_2023/mingyang-shi.markdown

```diff
@@ -0,0 +1,7 @@
+---
+name: Mingyang Shi
+short_name: Will
+github: beaioun
+---
+
+**Mingyang Shi** Hi, I am a Data Scientist/Engineer with AWS Solution Architect certified. My job is to deal with different sources of data and orchestrate a system and dashboard to show to the end users. I am activly seeaking new opportunities to improve myself and advance my career. I am glad you find my work meaningful! Thank you for visiting my page!
```

Comment:
nit: actively seeking*

Reply:
Sorry I'll recommit! A little bit rusty to type without grammarly.